docs/data-management/metrics/ratio-metric.md

```diff
@@ -3,15 +3,54 @@ sidebar_position: 2
+For example, consider the metric _average order value_. Typically, experiments are not randomized at the _order_ level because a user can have multiple orders. 
+Consequently, randomizing orders could lead to users experiencing multiple variants. When the experiment is randomized at the user level, it is not statistically valid to consider each
+_order_ an independent observation because orders from the same user are related. Treating the orders as statistically independent observations in experiments with user-based
+randomization typically leads to overconfident results (underestimating the variance). A proper estimation of the variance must recognize the fact that the observations (orders) are clustered
+at the user level. Although there are several valid approaches for estimating this variance, including [clustered standard errors](https://en.wikipedia.org/wiki/Clustered_standard_errors),
+Eppo uses the approach described in [Deng et al.](https://alexdeng.github.io/public/files/kdd2018-dm.pdf) and recommended in 
+Chapter 18 of _Trustworthy Online Controlled Experiments: A Practical Guide to A/B Testing_ by Ron Kohavi.
 
-Ratio metrics allow you to calculate ratio of two [simple metrics](/data-management/metrics/simple-metric), providing deeper insights into the relative performance of variations. This enables a more nuanced analysis of experimental results, allowing businesses to understand the impact of changes in a more comprehensive manner.
 
-For example, consider an _average order value_ metric, which is created by dividing revenue (sum of prices) by number of orders (_sum_ of items purchased or _count_ of prices).
+The approach is to redefine the metric of interest as a ratio of two simple metrics. For example, take _average order value_, which is defined by 
+dividing revenue (sum of prices) by the number of orders (sum of items purchased or count of prices).
+
 
 $$
 \text{Average order value} = \frac{\text{Revenue}}{\text{Number of orders}}
 $$
 
+We can multiply both the numerator and the denominator by the number of _users_ in the experiment:
+
+$$
+\text{Average order value} = \frac{\text{(Revenue)(Number of users)}}{\text{(Number of orders)(Number of users)}}
+$$
+
+which we can rearrange as follows:
+$$
+\text{Average order value} = \bigg(\frac{\text{Revenue}} {\text{Number of users}}\bigg) \bigg(\frac{\text{Number of users}} {\text{Number of orders}}\bigg)
+$$
+
+which can be expressed as the ratio of two simple metrics:
+$$
+\text{Average order value} = \frac{\text{Revenue per user}}{\text{Number of orders per user}}
+$$
+
+Both the numerator and the denominator are simple metrics whose analysis entity matches the randomization entity (a user). In general, the steps to define a ratio metric are
+as follows:
+1. Define the metric you wish to calculate and identify its analysis entity (e.g., an order in _average order value_).
+2. Divide both the numerator and the denominator by the _randomization_ entity (often a user).
```

Comment:
Eppo will automatically divide by the randomization entity, I think reads a bit like the user has to define this themselves

Reply:
Good callout. I made relevant updates here https://github.com/Eppo-exp/eppo-docs/commit/a1982eee5c58d85e6981dae8e1d0f2e87ea73a53, however based on our [Slack thread](https://eppo-group.slack.com/archives/C058QMZ6XQC/p1711399886470789) I don't feel confident that I fully understand how this is done. Let's sync in our 1x1